### PR TITLE
.NET 5.0.17 / 5.0.214 May 2022 Updates

### PR DESCRIPTION
**Pull request by lbussell**
Updates for .NET source-build May 2022 servicing.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -135,7 +135,10 @@ if [ "${internalPackageFeedPat:-}" ]; then
...
     {"endpoint":"https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-aspnetcore-5a129191/nuget/v3/index.json", "username":"optional", "password":"'$internalPackageFeedPat'"},
-    {"endpoint":"https://pkgs.dev.azure.com/dnceng/internal/_packaging/5.0.407-servicing.22171.19-shipping/nuget/v3/index.json", "username":"optional", "password":"'$internalPackageFeedPat'"},
+    {"endpoint":"https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-runtime-6a984143/nuget/v3/index.json", "username":"optional", "password":"'$internalPackageFeedPat'"},
+    {"endpoint":"https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-runtime-6a984143-1/nuget/v3/index.json", "username":"optional", "password":"'$internalPackageFeedPat'"},
+    {"endpoint":"https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-aspnetcore-02c6de4b/nuget/v3/index.json", "username":"optional", "password":"'$internalPackageFeedPat'"},
+    {"endpoint":"https://pkgs.dev.azure.com/dnceng/internal/_packaging/5.0.408-servicing.22219.28-shipping/nuget/v3/index.json", "username":"optional", "password":"'$internalPackageFeedPat'"},
```
Comment (lbussell) [inline build.sh:141]: I would like to clean up this list `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS` but I'm unsure of what can be removed without breaking the build. @crummel do you have any insight to this? Everything should be public, so could we potentially empty the whole list?

**Review by MichaelSimons — APPROVED** [review]

**Review by crummel — APPROVED** [review]

**Comment by lbussell** [comment]
The centos7 failure is CI-related and not product related - this is low priority to fix because 5.0 is EOL this month, so I will merge this.

**Pull request merged**